Distraction Shield/classes/BlockedSiteList.js

```diff
@@ -0,0 +1,88 @@
+
+function BlockedSiteList() {
+    this.list = [];
+
+    this.setList = function(blockedSiteArr) { this.list = blockedSiteArr; };
+    this.getList = function() { return this.list; };
```

Comment:
Could not find any instance where we did this, could you point me to and example? :)